### ISSUE TITLE
import from react/addons causing double React

### ISSUE THREAD

**Issue by Kilian** [chentsulin/electron-react-boilerplate#28]
I want to use CSSTransitionGroup, which is part of ReactWithAddons, but if I change my imports in the app folder to `import React from 'react/addons'` I get all kinds of errors because there are two React instances: one directly from filesystem, and one via webpack. 

It seems the one from filesystem (`[...]node_modules/react/lib/React.js:109`) is the one loaded by default and webpack additionally loads ReactWithAddons when i include `import React from 'react/addons'`. 

How can I make sure that just ReactWithAddons is used?

**Comment by chentsulin** [comment]
This should be fixed in `v0.2.9`. 

You can update `webpack/make-webpack-config.js` and try to confirm with below:

```
import React from 'react'
import ReactWithAddons from 'react/addons'

console.log('React and ReactWithAddons is using same react', React === ReactWithAddons)
// => true
```

**Comment by Kilian** [comment]
Works like a charm, thank you!
